contracts/WitnetRequestsBoard.sol

```diff
@@ -582,6 +582,16 @@ contract WitnetRequestsBoard is WitnetRequestsBoardInterface {
       _request.result.length == 0;
   }
 
+  /// @dev Ensures that rewards cover at least the gas cost needed for the complete DR life cycle within the Witnet Request Board.
+  /// @param _inclusionReward The amount for rewarding the reporting of the inclusion of the data request.
+  /// @param _tallyReward The amount for rewarding the reporting of the final result (aka tally) of the data request.
+  /// @param _blockReward The amount for rewarding the reporting of the blocks containing the DR inclusion and the DR result.
+  function isPayingGasCosts(uint256 _inclusionReward, uint256 _tallyReward, uint256 _blockReward) internal {
```

Comment:
In caase it fails, we could include the info about estimating the rewards by calling the estimageGas function

Reply:
Included